### PR TITLE
Add a banner photo to the website

### PR DESCRIPTION
**Pull request by ruthgrace**
I think that the website looks much more human and friendly when there's a picture of people doing stuff! This banner is the same one we use for Patreon, and the photos were taken by John B at Fixit Clinic at Noisebridge.

### CODE REVIEW

**Comment by ruthgrace** [comment]
Here's a screenshot of what the website looks like with the banner

website screenshot with photo banner

File: roles/mediawiki/templates/index.html [hunk 1]
```diff
@@ -191,6 +191,7 @@ <h1 class="title" style="text-indent: 1em">
       </div>
     </section>
 --></noscript>
+    <img src="https://i.imgur.com/1DDtmPW.jpg" alt="Photo of people fixing things at Noisebridge">
```
Comment (SuperQ) [inline roles/mediawiki/templates/index.html]: Let's add this file to the repo and push it to the server rather than use imgur.

**Comment by ruthgrace** [comment]
Sounds good! I wasn't sure how. Should I just make an img directory here?
How do I reference the file?

**Comment by SuperQ** [comment]
We've been doing this in the `mediawiki` role, maybe a good time to start spitting it into a `website` role.  From there it would be something like `/roles/website/files/foo.png`. Then we'd add a `copy` task to `/roles/website/tasks/main.yml`.

**Comment by ruthgrace** [comment]
Some questions I'm going to ask Victoria/Ben tomorrow:
- do i need the tasks/php.yml in my website role?
- is this permission OK for the image?
```-rw-r--r--@ 1 ruthgracewong  staff  364301 Mar  6  2018 Patreon 1600x400banner.jpg```
- am i doing this ansible thing right?

File: roles/website/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  tags:
+  - website
+
+- name: make image directory
+  file:
+    dest: "/srv/mediawiki/{{ mediawiki.domain }}/img"
+    state: directory
+    owner: www-data
+    group: www-data
+  tags:
```
Comment (SuperQ) [inline roles/website/tasks/main.yml]: We can probably drop the tags from this role.

**Comment by SuperQ** [comment]
Sorry, busy week.

**Pull request merged**